recipes/ucsc-genepredcheck/meta.yaml

```diff
@@ -1,57 +1,56 @@
 {% set package = "ucsc-genepredcheck" %}
 {% set program = "genePredCheck" %}
-{% set version = "447" %}
-{% set sha256 = "747a48486f7481d891e297baf63623b15d699265ede7339f654bcbc42481ac81" %}
+{% set version = "469" %}
+{% set sha256 = "3f95b26ccca854c965c05773481c269f2a0e6fb1906ae8a7fe2437c6ad543963" %}
 
 package:
-  name: {{ package }}
-  version: {{ version }}
+  name: "{{ package }}"
+  version: "{{ version }}"
 
 source:
   url: "http://hgdownload.cse.ucsc.edu/admin/exe/userApps.archive/userApps.v{{ version }}.src.tgz"
-  sha256: {{ sha256 }}
+  sha256: "{{ sha256 }}"
   patches:
-    - include.patch
-    - htmshell.patch
+    - "include.patch"
 
 build:
-  number: 2
-  skip: True # [osx]
+  skip: True  # [osx]
+  number: 0
   run_exports:
     - {{ pin_subpackage(package, max_pin=None) }}
-  ignore_run_exports:
-    - libpng
-    - libuuid
 
 requirements:
   build:
     - make
     - {{ compiler('c') }}
-    - {{ compiler('cxx') }}
-    - binutils # [linux]
   host:
     - libpng
     - libuuid
     - mysql-connector-c
-    - openssl
     - libopenssl-static
     - zlib
+
   run:
     - libpng
     - libuuid
     - mysql-connector-c
-    - openssl
+    - libopenssl-static
+    - zlib
 
 test:
   commands:
     # just check for existence and exec bit, because the individual packages have no unified behavior
-    - test -x $PREFIX/bin/{{ program }}
+    - test -x ${PREFIX}/bin/{{ program }}
 
 about:
-  home: "http://hgdownload.cse.ucsc.edu/admin/exe/"
-  license: "varies; see http://genome.ucsc.edu/license"
-  summary: "Validate genePred files or tables"
+  home: "https://hgdownload.cse.ucsc.edu/admin/exe/"
+  license: "Varies; see http://genome.ucsc.edu/license"
+  summary: "validate genePred files or tables"
+  dev_url: "https://github.com/ucscGenomeBrowser/kent"
+  doc_url: "https://github.com/ucscGenomeBrowser/kent/blob/master/README"
 
 extra:
-  additional-platforms:
-    - linux-aarch64
```

Comment:
@aliciaaevans Did this recipe fail to build on linux-aarch64 ?
It was building fine before. I will re-test it in a new PR!

Reply:
Here's the commit where I removed all the build failure files in favor of just removing the additional platform, if you want to look at the errors: https://github.com/bioconda/bioconda-recipes/pull/50393/commits/1ad838e3ab4a4670a4edaf16defbf5cce9e4709f but these all had a similar error. Maybe something changed in one of the dependencies since your original PR?

There are also a couple linux-64 with that same `undefined reference to knet_init_alt` error.  https://github.com/bioconda/bioconda-recipes/blob/bulk/recipes/ucsc-hubcheck/build_failure.linux-64.yaml

Reply:
I tried by restoring some of the changes in include.patch and meta.yaml at https://github.com/bioconda/bioconda-recipes/pull/50407 but it still fails the same way....